sklearn/linear_model/tests/test_passive_aggressive.py

```diff
@@ -125,6 +126,77 @@ def test_classifier_undefined_methods():
         assert_raises(AttributeError, lambda x: getattr(clf, x), meth)
 
 
+def test_class_weights():
+    # Test class weights.
```

Comment:
It would be great to reuse the SGD tests for PA (since they share the implementation). There'd be more work there, so I think this shouldn't be a show stopper for this PR. @amueller what do you think?